src/app/views/PersonDetail.svelte

```diff
@@ -57,11 +84,32 @@
     .qp({splits: [tagZapSplit(pubkey)]})
     .toString()
 
+  $: tabs = [
+    "notes",
+    "likes",
+    "collections",
+    "relays",
+    "following -" + $followsCount,
+    "followers " + $followersCount + "+",
+  ].filter(identity)
```

Comment:
Instead of interpolating the strings, use the `tab` slot in the `Tabs` component to customize the display. See Notifications.svelte for an example. You could also use the same badge style.